src/app/app.config.ts

```diff
@@ -53,7 +55,24 @@ export class AppConfig {
             });
         });
     }
+
+    /**
+     * Update once general css with values from config,
+     * this way we don't need to inject a style property in each element
+     * @param edition EditionConfig
+     */
+    updateStyleFromConfig(edition: EditionConfig) {
```

Comment:
I think it's not the right place for this feature.
Maybe you can also handle this in a better way

Reply:
I was requested to link the color style of the analogue and sources to the value placed in config for readingColorDark and readingColorLight.
I'm more than ok to change it but how would you do it otherwise?